global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+      "allowPrerelease": false
```

Comment:
Let me know what you think about this. The .net 8 prerelease was causing build issues. 

Seems like the nuget pack command wasn't putting the package where it was expected for acceptance tests to run

Reply:
Looks good.